src/nested-list.scss

```diff
@@ -0,0 +1,295 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+$block: #{$fd-namespace}-nested-list;
+
+@mixin fd-side-nav-arrow {
+  font-family: "SAP-icons";
+  font-style: normal;
+  font-weight: normal;
+  color: var(--sapList_TextColor);
+  font-size: var(--sapFontHeader4Size);
+  text-decoration: inherit;
+  text-transform: none;
+  speak: none;
+}
+
+@mixin fd-flex-center {
```

Comment:
This would probably better live in a shared utilities folder, as it's not particular to this file.

Reply:
Done